Subject: sched: guard struct sched_domain kABI changes
From: Mike Galbraith <mgalbraith@suse.de>
Date: Mon Aug 23 09:53:49 CEST 2010
Patch-mainline: never

sched: guard struct sched_domain kABI changes

silence kABI checker.

Signed-off-by: Mike Galbraith <mgalbraith@suse.de>
Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/linux/sched.h |    2 ++
 1 file changed, 2 insertions(+)

Index: linux-2.6.32-SLE11-SP1-TEST/include/linux/sched.h
===================================================================
--- linux-2.6.32-SLE11-SP1-TEST.orig/include/linux/sched.h
+++ linux-2.6.32-SLE11-SP1-TEST/include/linux/sched.h
@@ -1000,7 +1000,9 @@ struct sched_domain {
 	char *name;
 #endif
 
+#ifndef __GENKSYMS__
 	unsigned int span_weight;
+#endif
 	/*
 	 * Span of all CPUs in this domain.
 	 *
